### ISSUE TITLE
Retrieving Node binary location

### ISSUE THREAD

**Issue by mleegwt** [node-gradle/gradle-node-plugin#213]
To allow reporting code to Sonar we used:
```groovy
    def nodeExt = project.extensions.getByType(com.github.gradle.node.NodeExtension)
    def variantComputer = new com.github.gradle.node.variant.VariantComputer()
    def nodeDirProvider = variantComputer.computeNodeDir(nodeExt)
    def nodeBinary = variantComputer.computeNodeExec(nodeExt, variantComputer.computeNodeBinDir(nodeDirProvider)).get()
```
to determine the Node binary.
Since version 3.2.0 of the plugin this no longer works (I think it stopped working due to 98f7bdc22c9fccdd818aca1e0f18f64ba4891b95).  

I'm aware that the code snippet above is using the plugin internals. So I'm wondering whether there is a better way to retrieve the Node binary so we can correctly inform Sonar.
Thanks in advance!

**Comment by deepy** [comment]
I currently got a high fever from the booster but I think if you just change to `VariantComputer(PlatformHelper.INSTANCE)` it should work

The current roadmap is pretty much merging the old PRs, fixing the old bugs, fixing some more critical bugs, opening up the internal classes, and then starting to look into the worker api / shared build services.
Which I'm hoping will lead to a nicer way of passing the node binary to sonarqube

**Comment by mleegwt** [comment]
Works like a charm! Take care! Should this issue remain open for the 'nicer way'? Or is closing more appropriate.

**Comment by deepy** [comment]
This was fixed in `3.2.1`
